### ISSUE TITLE
Implement the force training after the calculation of the symmetric functions

### ISSUE THREAD

**Issue by mesonepigreco** [mesonepigreco/MINNIE#3]
Forces can be optimized after the calculation of the symmetric functions.
In fact, the loss function could be defined only in the neural network, with derivative with respect to the symmetric function instead that forces directly.

In this way, no long time required by the derivative of the symmetric function is taken and all the time goes into the training of the neural network.

This could lead to a massive speedup
